### ISSUE TITLE
NullReferenceException in DesktopService.PlatformTelemetry()

### ISSUE THREAD

**Issue by KirillOsenkov** [mono/monodevelop#4944]
On MonoDevelop startup on Windows I see a nullref:

```
 	MonoDevelop.Ide.IdeStartup.GetStartupMetadata Line 688	C#
>	MonoDevelop.Ide.IdeStartup.CreateStartupMetadata Line 339	C#
```

This is because `DesktopService.PlatformTelemetry();` returned null in `IdeStartup.cs` line 338 and line 339 doesn't check for `result` being null.

![image](https://user-images.githubusercontent.com/679326/40694893-84bb1260-6373-11e8-89a9-43a2a973d64d.png)

> VS bug [#623857](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/623857)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.6.0.987 (master)

Pull Request #4948 merged by: Lluis Sanchez
Author: mono
Commit: 164c66c38803098f28ed0de467f90befacd6e820 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.6.0.802 (master)

Pull Request #4948 merged by: Lluis Sanchez
Author: mono
Commit: 164c66c38803098f28ed0de467f90befacd6e820 (xamarin/monodevelop)
